app/views/social_networking/homes/show.html.erb

```diff
@@ -26,11 +26,15 @@
     <li class="list-group-item">
       <button class="btn btn-link" ng-click="home.newOnYourMindStatement()">What's on your mind?</button>
     </li>
-    <li class="list-group-item" ng-repeat="item in home.actionItems">
+    <li class="list-group-item" ng-repeat="item in home.actionItems" ng-if="home.actionItems.length > 0">
```

Comment:
It doesn't render the unordered list.  Instead https://github.com/cbitstech/social_networking/blob/msw_97540736_action_item_alert/app/views/social_networking/homes/show.html.erb#L34 is called.